tasks/script.js

```diff
@@ -28,6 +28,11 @@ module.exports = function(conf, base, outputName, input) {
     const duration = (end - start) / 1000;
     log(`Processed: ${input} → ${output} in ${duration} sec`);
   });
+  const cli = new CLIEngine({
+    useEslintrc: false,
+    configFile: conf.core.eslint
+  });
+  log(formatter(cli.executeOnFiles([input]).results));
```

Comment:
would be great to add the error tag to the log function only if there were errors